package.json

```diff
@@ -13,12 +18,17 @@
     "example": "examples"
   },
   "scripts": {
-    "generate-readme": "node bin/generate-readme.js",
-    "lint": "eslint lib examples bin",
-    "pretest": "npm run lint",
-    "test": "jest",
-    "coverage": "jest --coverage",
-    "start": "rackt server"
+    "generate-readme": "scripty",
```

Comment:
It's sort of a shorthand for pointing to individual shell scripts, with some added benefits like being able order multiple targets under one umbrella (e.g. "build", "build:component", "build:examples"). The only downside (as I see it) is that `npm run` becomes less helpful if you're not familiar with scripty (as you pointed out).